sources/packages/web/src/views/aest/student/applicationDetails/ApplicationRestrictionsManagement.vue

```diff
@@ -9,18 +9,25 @@
         }"
         subTitle="Application Restriction Management"
       />
-      <application-header-title :application-id="applicationId" />
+      <application-header-title
```

Comment:
As the outcome of our discussion, v-if was an outcome. As per my discussion with @andrewsignori-aot I recollected the watchEffect that we used.  
![image](https://github.com/user-attachments/assets/bef78bb5-d68c-4ce2-aa93-6e93f397ff6e)

Reply:
I would still consider this as not a blocker. v-if should be ok for the initial implementation.